src/DbTarget.php

```diff
@@ -6,11 +6,10 @@
 
 use RuntimeException;
 use Throwable;
-use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Db\Driver\PDO\ConnectionPDOInterface;
```

Comment:
Yes remove `getDsn` from https://github.com/yiisoft/log-target-db/blob/b2dbe258d3f52810ac77e41c5f3838ca63be8245/src/Migration/M202101052207CreateLog.php#L41, since each connection can only have one table with the same name.

Reply:
done.